test/integration/vaultdynamicsecret_integration_test.go

```diff
@@ -662,13 +663,69 @@ func TestVaultDynamicSecret_vaultClientCallback(t *testing.T) {
 	}
 
 	tests := []struct {
-		name     string
-		create   int
-		expected map[string]int
+		name        string
+		create      int
+		triggerFunc func(t *testing.T, reconciledObjs []*secretsv1beta1.VaultDynamicSecret)
+		expected    map[string]int
```

Comment:
Looks like `expected` isn't used. Or do we have plan for using it in later work?